R/zzz.R

```diff
@@ -12,22 +12,23 @@ namespaces_reset <- function() {
 
 namespaces_default <- function() {
   has_glue <- requireNamespace("glue", quietly = TRUE)
+  is_running_gha <- isTRUE(Sys.getenv("GITHUB_ACTIONS") == "true") && !needs_stdout()
```

Comment:
I need to think about this one, as I can imagine (and know) folks using `logger` in GHA writing outside of the GHA logs, although hopefully explicitly setting the layout as well :thinking: 

Anyway, this is a super cool idea, thank you! I'm traveling/teaching in the next ~48 hours, but I will definitely get back to this PR for a more detailed look on Friday.

Reply:
Yeah, I can imagine. Ultimatively it is your call and I understand being weary of messing up peoples current deployment. On the other hand it would be a nice quality-of-life improvement for new projects